Vagrantfile

```diff
@@ -0,0 +1,53 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+release       = ENV['RELEASE'] ? ENV['RELEASE'] : 'stable'
+hostname      = ENV['HOSTNAME'] ? ENV['HOSTNAME'] : 'st2vagrant'
```

Comment:
`st2-packages`, better to hardcode it, don't need to pass it via `HOSTNAME` env.